app/src/pages/playground/InvocationParametersFormFields.tsx

```diff
@@ -31,11 +28,29 @@ const InvocationParameterFormField = ({
   field,
   value,
   onChange,
+  errors,
+  control,
 }: {
   field: InvocationParameter;
   value: unknown;
   onChange: (value: unknown) => void;
+  errors: FieldErrors<Record<string, unknown>>;
+  control: Control<Record<string, unknown>>;
 }) => {
+  const invocationName = field.invocationName;
+  if (!invocationName) {
+    throw new Error("Invocation name is required");
+  }
+  const errorMessage = errors[invocationName]?.message;
+  const required = field.required
+    ? `${field.label || invocationName} is required`
+    : undefined;
+  const min = field.minValue
+    ? `${field.label || invocationName} must be at least ${field.minValue}`
+    : undefined;
+  const max = field.maxValue
```

Comment:
probably need to do an actual loose null check here in case the min or max is 0

Reply:
Good catch, thanks cursor 🙄

Reply:
Fixed